src/material-experimental/mdc-form-field/_mdc-text-field-theme-variable-refresh.scss

```diff
@@ -0,0 +1,63 @@
+@use 'sass:color';
+@use '@material/theme/variables' as theme-variables;
```

Comment:
As mentioned in the meeting. The variable value expressions are basically just copied over from their `variables.scss` file. I didn't bother updating every function invocation / variable reference to use the old syntax. Since the MDC styles already use `@use` and `@forward` anyway, I don't think there is any reason why we can't use it here (and save some work)